main/tests/test_string.h

```diff
@@ -31,12 +31,1009 @@
 #ifndef TEST_STRING_H
 #define TEST_STRING_H
 
+#include <thirdparty/doctest/doctest.h>
+
+#include "core/io/ip_address.h"
 #include "core/os/main_loop.h"
+#include "core/os/os.h"
 #include "core/ustring.h"
 
+#include "modules/regex/regex.h"
```

Comment:
This should be conditional to the regex module being enabled, as it was previously.

Reply:
Fixed